share/picongpu/tests/XrayScattering/etc/picongpu/1.cfg

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2020 Axel Huebl, Rene Widera, Felix Schmitt,
+#              Pawel Ordyna
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="0:30:00"
+
+TBG_devices_x=1
+TBG_devices_y=1
+TBG_devices_z=1
+
+TBG_gridSize="120 120 32"
```

Comment:
see below - 32 cells used in z but 2D simulation setup - what is intended?

Reply:
I think for 2D this 32 for z grid size is just skipped. However I would agree, if the example is strictly for the 2D case, better remove for better clarity.

Reply:
However from the `cmakeFlags` file it seems the 3D case is also tested.

Reply:
Yes, I intended to look at the 3D case as well.